playwright-e2e/pages/pancan/enrollment/consent-form-page.ts

```diff
@@ -34,30 +52,61 @@ export default class ConsentFormPage extends PancanPage {
    * <br> Type: Input
    */
   firstName(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:CONSENT_FIRSTNAME' });
+    return new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.firstName });
```

Comment:
Oh nice--so the `Patient` object encapsulates which specific fields to hit for these minor variations.  That saves us from having to duplicate page classes.